pizzeria-backend/Models/AddOnEntity.cs

```diff
@@ -9,10 +11,16 @@ public class AddOn : IAddOn
         public int Id { get; set; }
         public string Name { get; set; }
         public string Description { get; set; }
+        [ForeignKey("Category")]
+
         public int CategoryId { get; set; }
         [JsonIgnore]
         public AddOnsCategory Category { get; set; }
         public decimal Price { get; set; }
         public int AmountInGrams { get; set; }
+
+        [JsonIgnore]
+        [IgnoreDataMember]
```

Comment:
What does JsonIgnore and IgnoreDataMember mean?

Reply:
The annotations were used so the member of the class is not serialized. Probably not needed in the current implementation